src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml

```diff
@@ -3,7 +3,26 @@
 @model MemberActivitiesViewModel;
 
 <div>
-    <h3 class="govuk-heading-m">Upcoming events (@Model.FutureEventsCount)</h3>
+    <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-2">@Model.FirstName's activities</h1>
+    <p>Last event sign up: @Model.LastEventSignUpDate</p>
+    <h3 class="govuk-heading-m">
+        In the past 12 months
+    </h3>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half">
+            <div class="app-figure">
+                <p class="govuk-body app-figure__figure">@Model.EventsAttendedCount</p>
+                <p class="govuk-body app-figure__label">Events attended</p>
```

Comment:
"Events attended" or "attended events" content is inconsistent on the ticket, worth double checking

Reply:
We normally follow figma when there is contradiction between the ticket and figma.